extensions/src/ACRE2Mumble/MumbleCallbacks_init.cpp

```diff
@@ -51,6 +49,7 @@ void mumble_onServerSynchronized(mumble_connection_t connection) {
 
     // set ID on every new connection
     acre::id_t clientId = 0;
+	// TODO: Missing error handling of API call
```

Comment:
:point_up:

Reply:
More error handling can probably done in multiple API call locations.